backend/testfiles/execution/dict.tests

```diff
@@ -16,6 +16,7 @@ Dict.fromListOverwritingDuplicates_v0 [(1,1)] = Test.typeError_v0 "Expected `key
 Dict.fromListOverwritingDuplicates_v0 [("a",1); 2; ("c",3)] = Test.typeError_v0 "All list items must be `(key, value)`"
 Dict.fromListOverwritingDuplicates_v0 [Test.typeError_v0 ""] = Test.typeError_v0 ""
 
+// CLEANUP the first test here feels surprising - should it error or something?
 Dict.fromList_v0 [("duplicate_key",1); ("b",2); ("duplicate_key",3)] = Nothing
```

Comment:
open question. I find the result here confusing

Reply:
I would find a result more obvious, but given the only failure mode is overlapping keys, I'm not sure.